code/object/object.cpp

```diff
@@ -574,7 +588,7 @@ void obj_delete(int objnum)
 	case OBJ_START:
 	case OBJ_WAYPOINT:
 	case OBJ_POINT:
-		Assert(Fred_running);
+		//Assert(Fred_running);
```

Comment:
This should be removed or explained in some way.

Reply:
The assert needed to be removed because trying to delete all objects in a mission will, by necessity, hit point objects; an assertion that FRED is running is therefore not needed at that point.

Reply:
That makes sense but it should be actually removed and not just commented out.

Reply:
It will be in the next commit to this branch